### ISSUE TITLE
save skin functionality

### ISSUE THREAD

**Issue by distributev** [distributev/the-app#8]
When I first tested this after git pull I got the impression that the skin was not saved during logout and login again. 2nd time it worked ==> test and double check this functionality works all the time.

Another thing is that the saved skin should be applied even after the user is doing logout and he comes again to the login screen. The login screen should have the user's saved theme (from local storage - in case the user did not delete browser cache).

**Comment by nabilzhafri** [comment]
hello. please note that this functionality should be included in some other repo, like we discussed before. (it's the issue we've been talking about before) the reason its not working at the moment, because the server code doesn't seem to support it yet.

**Comment by nabilzhafri** [comment]
and note that referring to previous implementation, it doesn't save into cache, but it's saved into database. I simply copied existing code from the other repo when saving theme.

**Comment by distributev** [comment]
I understand your points. 

However after I tried 2nd time it was looking like the sking was saved. How was that?
And I remember, at the time the project was done, I tested myself on the original repo the functionality to save the theme and it was saving fine. 

P.S - unless I miss something obvious.

**Comment by nabilzhafri** [comment]
Reason because the server code to save theme is not enabled yet for this repo. Remember that you have other repos previously, and I only copied admin lte to this repo.

Let me just enable it for you by adding the server code.

**Comment by distributev** [comment]
Thank you.

It means the other repo managed to "get away" without fully completing their task. This is strange since I tested and saving of the theme was working there. I don't know how I can see the save working is the server code isn't there. It's possible I could have missed something during my testing.

**Comment by nabilzhafri** [comment]
You're welcome :) no worry, consider it as a favor form me. I implemented from scratch, plus I still has to update theme implementation since I replaced sequalize with bookshelf.

Anyway, saving theme functionality is now in master. check it out :)

**Comment by distributev** [comment]
Thank you very much.

I was not aware that server code is missing. You tried to tell me few days ago but I'm perplexed I actually tested the functionality and it was looking as working. I now believed somehow my testing was mis-leading / wrong since it couldn't work properly without server code. 

Thanks again anyway.

P.S I'm tempted to pull again original repository and retest to see what the hell I tested at that time.
